hawkbit-dmf/hawkbit-dmf-amqp/src/test/java/org/eclipse/hawkbit/integration/AbstractAmqpServiceIntegrationTest.java

```diff
@@ -87,6 +94,7 @@ public void initListener() {
         replyToListener = harness.getSpy(ReplyToListener.LISTENER_ID);
         assertThat(replyToListener).isNotNull();
         Mockito.reset(replyToListener);
+        replyToListener.purge();
```

Comment:
Shouldn't you call this before you reset the spy. Otherwise this call is already counted.